### PR TITLE
Adding readme instructions to execute the service.

### PR DESCRIPTION
**Pull request by sflechas**

### CODE REVIEW

**Comment by nisargthakkar** [comment]
@sflechas Can you also add more instructions on running the tests, and getting the coverage? Prof @rofrano 's repo at https://github.com/nyu-devops/lab-flask-rest is a good example.

File: README.md [hunk 1]
```diff
@@ -15,3 +15,14 @@ GET /wishlists | LIST | Show all wishlists
 GET /wishlists?q=querytext | QUERY | Search for a wishlist
 GET /wishlists/`<id>`?q=querytext | QUERY | Search for items in wishlist
 PUT /wishlists/`<id>`/items/`<itemid>`/add-to-cart | ACTION | Move an item from wishlist to the shopping cart
+
+To execute the service run:
+
+```
+vagrant up --provision
+vagrant ssh
+cd /service
```
Comment (nisargthakkar) [inline README.md]: This will error out. We only need `/vagrant`

File: README.md [hunk 2]
```diff
@@ -15,3 +15,14 @@ GET /wishlists | LIST | Show all wishlists
...
+FLASK_APP=service:app flask run -h 0.0.0.0
```
Comment (nisargthakkar) [inline README.md]: Do we not need to specify a port?

**Comment by sflechas** [comment]
Yes, sure, I'll add them.  
I thought you had added those already, what happened to them?

File: README.md [hunk 3]
```diff
@@ -15,3 +15,14 @@ GET /wishlists | LIST | Show all wishlists
...
+```
+
+Then open a browser session to http://0.0.0.0:5000/
```
Comment (nisargthakkar) [inline README.md]: I'm not sure if `0.0.0.0` is correct. I think it should be `localhost` or `127.0.0.1`. It doesn't work on Windows. Not sure about Mac/Linux.

**Review by nisargthakkar — CHANGES_REQUESTED** [review]

**Comment by nisargthakkar** [comment]
I closed that PR since the README updated a bit after that and that version would have messed up some content

File: README.md (hunk 1 above)
Comment (sflechas) [reply README.md]: No, it needs both, these were the instructions that I had in my local branch. /vagrant only won't work either.  These instructions are no longer valid.  
I am updating my local copy with all the changes and submit a new update later today.

File: README.md (hunk 2 above)
Comment (sflechas) [reply README.md]: Not needed to run the service, only on the URL

File: README.md (hunk 3 above)
Comment (sflechas) [reply README.md]: That is how It works in my Mac, how does it work for you?

File: README.md (hunk 1 above)
Comment (nisargthakkar) [reply README.md]: Okay. Thanks

File: README.md (hunk 2 above)
Comment (nisargthakkar) [reply README.md]: Okay. Thanks

File: README.md (hunk 3 above)
Comment (nisargthakkar) [reply README.md]: Going to `http://0.0.0.0:5000/` says `This site can’t be reached`. I need to put `http://localhost:5000/` or `http://127.0.0.1:5000/` to get to the service. Probably this is just a Windows thing

Comment (sflechas) [reply README.md]: localhost also works on my Mac, so I'll leave it with localhost.

**Review by nisargthakkar — APPROVED** [review]
LGTM

**Pull request merged**